trio/_threads.py

```diff
@@ -100,7 +101,7 @@ def run(self, afn, *args):
           Cancelled: if the corresponding call to :func:`trio.run` completes
               while ``afn(*args)`` is running, then ``afn`` is likely to raise
               :class:`Cancelled`, and this will propagate out into
-          RuntimeError: if you try calling this from inside the Trio thread,
+          RuntimeError: if you try calling this from inside a Trio thread,
```

Comment:
hmm. I see the reasoning here (you can have more than one Trio thread in a process) but I think that's quite an expert-level trick, and being consistent about calling it "the Trio thread" might be more understandable since for 99% of users there will only be one. (Once you have multiple threads you start needing to worry about which thread owns each channel/lock/etc, and you need to write your own, uhh, "NonBlockingTrioPortal" to communicate between them.)

I think my preference here would be to be consistent about saying "the Trio thread" in our docs, and maybe add a note somewhere that yes, you can have multiple threads in the same process each in their own call to `trio.run()`, but communicating between those threads will require dipping down into hazmat. Not a strong preference though.

Reply:
My preference too.